courses/machine_learning/deepdive2/structured/solutions/3c_bqml_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,1152 @@
+{
```

Comment:
<p>Retrieve the evaluation table from either a local variable or a BQ table saved by the previous cell to avoid recomputing the evaluation just to take the square root of the mse.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Same</p>